lib/verbalize/action.rb

```diff
@@ -17,8 +17,19 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
+        # treat `input` as an accessor if nothing is passed in.
+        return @inputs || [] if required_keywords.empty? && optional.empty?
```

Comment:
I wonder if we should just make a separate `required_arguments` accessor method.  It's a bit unintuitive that `inputs` takes both required and optional inputs, but only returns one of the two.  I would prefer if `inputs` returned everything instead of separating them out, or if we used a `required_arguments` accessor method.  Or, we can do all three - `inputs` returns all of them as a hash, `(required|optional)_inputs` returns them separately.